src/pocketmine/Player.php

```diff
@@ -2980,6 +2980,14 @@ public function sendTip($message){
 		$pk->message = $message;
 		$this->dataPacket($pk);
 	}
+	
+	public function whisper($from, $message){
+		$pk = new TextPacket();
+		$pk->type = TextPacket::TYPE_WHISPER;
+		$pk->source = $from instanceof Player ? $from->getDisplayName() : $from;
```

Comment:
I was actually referring to the possibility that the sender parameter could be something else _apart_ from a Player or string. Supplying such an invalid object would potentially cause a fatal crash